editor support: add neovim (#447)
melange-re/melange: README.md

}
```

> #### Neovim >= 0.8 to get built-in LSP support

Reviewer: ```suggestion
#### Neovim (Built-in LSP support available on version >= 0.8)
```

```lua

> require'lspconfig'.ocamllsp.setup{

Reviewer: ```suggestion
require('lspconfig').ocamllsp.setup {
```

}
```

> #### Neovim >= 0.8

Reviewer: please explain why e.g. "to get built-in LSP support"

> ```lua

Reviewer: This is doing too much. The configuration shouldn't be more than a few lines:

```lua
require('lspconfig')['ocamllsp'].setup {
  cmd = { 'ocamllsp', '--fallback-read-dot-merlin' },
}
```

Setting up the environment is a user problem, e.g. start `nvim` with `esy nvim`, `nix develop #. -c nvim`, `opam exec -- nvim`, etc.
